Library/Formula/unrar.rb

```diff
@@ -15,6 +15,11 @@ class Unrar < Formula
   def install
     system "make"
     bin.install 'unrar'
+    system "make", "clean"
```

Comment:
Ok, can you add a Ruby comment when you've sent the email just detailing (briefly) when you sent it, to whom and what you said. Thanks!